apps/web/src/pages/templates/components/LackIntegrationAlert.tsx

```diff
@@ -50,18 +50,34 @@ export function LackIntegrationAlert({
     }
   };
 
+  function handleErrorRectangleClick() {
+    if (isPrimaryMissing) {
+      openSelectPrimaryIntegrationModal({
+        environmentId: environment?._id,
+        channelType: channelType,
+        onClose: () => {
+          segment.track(TemplateEditorAnalyticsEnum.CONFIGURE_PRIMARY_PROVIDER_BANNER_CLICK);
+        },
+      });
+    } else {
+      openIntegrationsModal(true);
+      segment.track(TemplateEditorAnalyticsEnum.CONFIGURE_PROVIDER_BANNER_CLICK);
```

Comment:
After merging from next, there are now 2 of these functions :) Please remove one of them and we will merge🚀